Formula/platformio.rb

```diff
@@ -1,10 +1,10 @@
 class Platformio < Formula
   include Language::Python::Virtualenv
 
-  desc "Open-source ecosystem for embedded development"
+  desc "Professional collaborative platform for embedded development"
```

Comment:
Is it no longer open-source?

Reply:
It’s open source, just better explanation what is PlatformIO.